gpiozero/output_devices.py

```diff
@@ -517,6 +583,22 @@ def blink(
         :param float fade_out_time:
             Number of seconds to spend fading out. Defaults to 0.
 
+        :param float on_value:
+            value during on_time.
+            Defaults to 1 (full on).
+
+        :param float off_value:
+            value during off_time.
+            Defaults to 0 (full off).
+
+        :type end_value: float or None
+        :param end_value:
+            value to set after a full uninterrupted blink.
+            Defaults to None, on which no value will be set.
+
+        :param int fps:
```

Comment:
Why add this as an extra parameter? Wouldn't it make more sense to just (internally) use `self.frequency` ?

Reply:
It would be possible to use self.frequency. I borrowed this from the _blink_device, where it was a default parameter, but never set on the call.
fps defines, how smooth the transition of the fade will be. 
A value of 25 means, that every 40 ms a new value will be set.  
On a PWM frequency of 100 Hz that means every 4th zero crossing the pulse width will be changed. That is a heavy load already. In most cases a lower value is enough for a smooth eye pleasing transition.